ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -514,6 +514,107 @@ In AVD 4.0.0, we had "switched" as the default value for `ethernet_interfaces[].
 
 With AVD 5.0.0, the default value for `type` in `ethernet_interfaces` and `port_channel_interfaces` is no longer supported. The `type` key must now be explicitly defined in the input variables if it is needed in the configuration and documentation.
 
+### Removal of `type` key dependency for rendering ethernet/port-channel interfaces configuration and documentation
+
+#### `vlan_id` configuration
+
+In AVD 4.0.0, the configuration for `vlan_id` in ethernet interfaces and port-channel interfaces was rendered only if `ethernet_interfaces[].type` or `port_channel_interfaces[].type` was set to `l2dot1q`.
+
+With AVD 5.0.0, the dependency on the `type` key for rendering the configuration of `vlan_id` in ethernet and port-channel interfaces has been removed.
+
+```diff
+-type: l2dot1q
+vlan_id: 10
+```
+
+#### `switchport` configuration and documentation
+
+In AVD 4.0.0, the configuration and documentation for `switchport` in ethernet and port-channel interfaces were renderend only if `ethernet_interfaces[].type` or `port_channel_interfaces[].type` was set to `switched` along with the keys `mode`, `vlans`, `native_vlan`, `native_vlan_tag`, `trunk_groups`, `phone`, `vlan_translations`, `trunk_private_vlan_secondary`, `pvlan_mapping`.
+The `no switchport` command was renderend if `ethernet_interfaces[].type` or `port_channel_interfaces[].type` was set to `routed`.
+
+With AVD 5.0.0, a new key for `switchport` has been introduced under ethernet and port-channel interfaces data-model, which generates the configuration and documentation for `switchport` independent of the `type` key.
+The keys `mode`, `vlans`, `native_vlan`, `native_vlan_tag`, `trunk_groups`, `phone`, `vlan_translations`, `trunk_private_vlan_secondary`, `pvlan_mapping` are deprecated, to be removed in AVD 6.0.0.
+
+```diff
+-type: switched
+-mode: trunk
+-vlans: '1-5'
+-native_vlan: 20
+-native_vlan_tag: true
+-trunk_groups: ['group1', 'group2']
+-trunk_private_vlan_secondary: true
+-pvlan_mapping: 20-30
+-  phone:
```

Comment:
something looks wrong with the context of this key. Are we missing something above phone?